exotic.py

```diff
@@ -1735,8 +1526,10 @@ def realTimeReduce(i):
         print('*******************************************')
         print("Planetary Parameters for Lightcurve Fitting")
         print('')
+        
+        # based on the logic above this will never be false...
+        if not CandidatePlanetBool: 
```

Comment:
It will be false if the planet cannot be found in the NASA Exoplanet Archive (i.e., a TESS TOI).

Reply:
Added back in, that comment was from an old iteration.